client/src/assets/stylesheets/App.css

```diff
@@ -0,0 +1,3 @@
+.hero {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 0.25)), url("../images/homepage-hero.png");
```

Comment:
could this be done with Tailwind? similar to how the login background image is

Reply:
Good call, I will fix that!